frontend/src/collapsible.js

```diff
@@ -0,0 +1,21 @@
+/*!
+ * sri sri guru gaurangau jayatah
+ */
+
+window.collapsible = () => {
+  return {
```

Comment:
Here we can shorten the syntax even more because the only clause of the function body is `return`. See [the documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#:~:text=To%20return%20an%20object%20literal%20expression%20requires%20parentheses%20around%20expression) and [`player.js`](https://github.com/PureBhaktiArchive/archive-web-app/blob/b090312f828b6d822d58e4a776ce77ea829116ef/frontend/src/player.js#L7) for example.

Reply:
This is left unfixed. It's quite easy to do because VSCode even suggests this:

![image](https://user-images.githubusercontent.com/6065071/130364515-bc349e84-dc4a-4ffe-b914-86c6b3f74441.png)

I will add a commit myself then.